doc/AzureDocument.md

```diff
@@ -0,0 +1,197 @@
+# Accelerate real-time big data analytics with Spark connector for Azure SQL Databases and SQL Server
+
+The Spark connector for Azure SQL Databases and SQL Server enables SQL databases, including Azure SQL Databases and SQL Server, to act as input data source or output data sink for Spark jobs. It allows you to utilize real time transactional data in big data analytics and persist results for adhoc queries or reporting. Comparing to the built-in JDBC connector, this connector provides the ability to bulk insert data into SQL databases. It can outperform row by row insertion with 10x to 20x faster performance. The Spark connector for Azure SQL Databases and SQL Server also supports AAD authentication. It allows you securely connecting to your Azure SQL databases from Azure Databricks using your AAD account. It provides similar interfaces with the built-in JDBC connector. It is easy to migrate your existing Spark jobs to use this new connector.
```

Comment:
In the first sentence, Azure SQL Databases should be singular, not plural and, in the second half of the sentence, should be ", including databases in Azure SQL Database and SQL Server"

Also, "Comparing to the" should be "Compared to the"

Similarly, further on, should be Azure SQL Database - not plural